Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+RUN cp ./files/config.py ./mote/config.py
```

Comment:
What if there's already a modified config.py in the `mote` directory modified by the user, It'll copy the contents of the base again, I think, for now, it would be good to remove it.

Reply:
Would not be much harm to do so right? As the changes were already made to the file, it would just copy that stuff over but if we remove the 5th line - we make them copy that file again by themselves.